source/jsonnet/england-wales/ccs/blocks/who-lives-here/who_lives_here_interstitial.jsonnet

```diff
@@ -15,13 +15,13 @@ local placeholders = import '../../../lib/placeholders.libsonnet';
         },
       },
       {
-        description: 'A <em>household</em> is one person living alone, or a group of people (not necessarily related), who share cooking facilities, and share a\n        living room, <em>or</em> sitting room, <em>or</em> dining area.',
+        description: 'A <em>household</em> is one person living alone, or a group of people (not necessarily related), who share cooking facilities <em>and</em> share a\n        living room, <em>or</em> sitting room, <em>or</em> dining area.',
```

Comment:
Shouldn't this be strong, not em?

Reply:
I used em cause it was already there in the rest of the sentence, but have updated them all to be strong now.